specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -2401,6 +2405,15 @@
       "format": "uuid",
       "x-ms-parameter-location": "method",
       "collectionFormat": "csv"
+    },
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).",
```

Comment:
I am a little concern about the `Endpoint` setting. Previously, we also have `Endpoint` concept in Windows SDK [1] and many other places where the endpoint is the whole base url including the path `/face/v1.0`. I prefer we keep consistency and that is also more general since the path dose not need to be fixed. But if this is a team decision that we will keep the same style in all cognitive services, I am OK with it.

[1] https://github.com/Microsoft/Cognitive-Common-Windows/blob/10b0875f4f984186db45ec6da987617dd967d14f/SampleUserControlLibrary/SubscriptionKeyPage.xaml#L49

Reply:
Hi @huxuan, the decision is a compromise according to the feedback from multiple teams as well as the limitations of swagger. (In fact we have this change because we prefer consistency over back compatibility)

In this PR we changed the definition of Endpoint, we are planning to update `Endpoint` in documents and Azure Portal when all the teams agree with the change.

Reply:
@huxuan BTW feel free to bring up you concerns in the e-mail thread.